monailabel/tasks/infer/basic_infer.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         output_label_key: str = "pred",
         output_json_key: str = "result",
         config: Union[None, Dict[str, Any]] = None,
-        load_strict: bool = False,
+        load_strict: bool = True,
```

Comment:
lets keep the base class value same.. lets not change.. we override it to true in all derived classes